src/common/TonesOverTime.js

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useContext, useState } from 'react';
+import { Line } from 'react-chartjs-2';
+import UserContext from '../modules/Context/UserContext';
+import * as API from '../API/APIcalls';
+
+import fakeDreams from '../data/fakeDreams';
+import fakeUser from '../data/fakeUser';
+
+const TonesOverTime = () => {
+  const user = useContext(UserContext);
+  const [allDreams, setAllDreams] = useState(null);
+  const [chartDayCount, setChartCount] = useState(70);
+  const [chartDates, setChartDates] = useState(null);
+  const [chartTones, setChartTones] = useState([]);
+  const [chartPlotDatasets, setChartPlotDatasets] = useState([]);
+
```

Comment:
These states can be grabbed to also provide data for over tones, etc. Still need to build out a little of that functionality though.